composer.json

```diff
@@ -26,22 +26,27 @@
         }
     },
     "require": {
-        "php": "^5.5 || ^7.0",
-        "zendframework/zendframework": "^2.5.3",
-        "zfcampus/zf-apigility": "~1.1",
-        "zfcampus/zf-apigility-documentation": "^1.0.5",
-        "zfcampus/zf-development-mode": "~2.0"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-component-installer": "^1.0 || ^0.3 || ^1.0.0-dev@dev",
+        "zfcampus/zf-apigility": "^1.3",
+        "zfcampus/zf-apigility-documentation": "^1.2.1",
+        "zfcampus/zf-development-mode": "^3.0"
     },
     "require-dev": {
-        "zendframework/zftool": "dev-master",
-        "zendframework/zend-developer-tools": "dev-master",
-        "zfcampus/zf-apigility-admin": "^1.4.1",
-        "zfcampus/zf-apigility-admin-ui": "^1.2.3",
-        "zfcampus/zf-deploy": "~1.0"
+        "zendframework/zend-developer-tools": "^1.1",
+        "zfcampus/zf-apigility-admin": "^1.5",
+        "zfcampus/zf-asset-manager": "^1.0",
+        "zfcampus/zf-deploy": "^1.2"
     },
     "suggest": {
-        "zfcampus/zf-apigility-doctrine": "zfcampus/zf-apigility-doctrine ~1.0 to create Doctrine-Connected REST services",
-        "zfcampus/zf-http-cache": "zfcampus/zf-http-cache ~1.0 to add HTTP caching to your API",
-        "zfr/zfr-cors": "zfr/zfr-cors ~1.2 to add CORS support to your API"
+        "zfcampus/zf-apigility-doctrine": "zfcampus/zf-apigility-doctrine ^1.0 to create Doctrine-Connected REST services",
```

Comment:
`zfcampus/zf-apigility-doctrine` is alredy in version 2, hopefully soon will be `2.1`